Dockerfile-gpu

```diff
@@ -14,8 +14,6 @@ COPY ./deployment/append_memory_setting.sh /home/model-server/append_memory_sett
 RUN chmod +x /home/model-server/append_memory_setting.sh \
     && /home/model-server/append_memory_setting.sh
 
-COPY handler*.py ./
```

Comment:
Do we not need the handler now?

Reply:
we do but not here. It's included in the Dockerfile-build step. then it is contained in the .mar that container creates and it gets copied over to these services.